model_definition.md

```diff
@@ -249,6 +257,9 @@ We use informative priors for parameters that have been well characterized in th
 | Initial exponential growth rate | $r \sim \mathrm{Normal}(0, 0.01)$ | Chosen to assume flat dynamics prior to observations |
 | Infection feedback term | $\gamma \sim \mathrm{logNormal}(6.37, 0.4)$ | Weakly informative prior chosen to have a mode of 500 in natural scale, based on posterior estimates of peaks from prior seasons in a few jurisdictions |
 | Day of the week effects | $\frac{\vec{\omega}}{7} \sim \mathrm{Dirichlet}(5, 5, 5, 5, 5, 5, 5)$ | Weakly informative prior with a mode at even daily reporting (no effects) |
+| Standard deviation in  the log of the site-lab level multiplier $M_{ij}$ | $\sigma_m \sim \mathrm{Normal}(0, 0.25) $| Weakly informative prior chosen to allow average magnitude of  concentrations to deviate |
+| Standard deviation in the log of the mode of the standard deviation between site-lab level standard deviations| $\log(mode_{\sigma}) \sim \mathrm{Normal}(1,1)$ | Weakly informative prior chosen to allow site-lab level standard deviations to vary |
```

Comment:
Not the log of the mode, just the mode. And it's just the modal observation error s.d.

```suggestion
| Mode of the site-level observation standard deviation | $mode_{\sigma} \sim \mathrm{Normal}(1,1)$ | Weakly informative prior chosen to allow site-lab level standard deviations to vary |
```